test/openshift/e2e/sequential/1-018_validate_disable_default_instance/03-check.yaml

```diff
@@ -3,3 +3,6 @@ kind: TestStep
 timeout: 1200
 error:
 - errors.yaml
+- cluster-roles-errors.yaml
+- roles-errors.yaml
+- service-accounts-errors.yaml
```

Comment:
@anandf I think it may be easier to replace `03-check.yaml` with `03-errors.yaml` and combine all these files in 1 place
WDYT?

Reply:
I have kept the file name as it was earlier `03-check.yaml` Not sure if this is due to some naming convention followed for kuttl. I have just added additional files to fail the tests if those RBAC resources are present.